### DISCUSSION TITLE
Flood mediainfo show error (qbittorrent)

### DISCUSSION THREAD

**Discussion by lilws** [jesec/flood#173] [Q&A] [answered]
I setup docker image jesec/flood and successful connect to linuxserver/qbittorrent too. It worked fine until recently I tried to check for mediainfo and see this error: 

<img width="695" alt="Screen Shot 2021-01-27 at 16 03 28" src="https://user-images.githubusercontent.com/12478478/105968558-ced1bc00-60b9-11eb-93dc-7eb8a7cd75dd.png">

Is there anyway to fix this?

I recently use ibryzg/rutorrent-flood image and good so far, only one thing let me down is the stats of rtorrent is not saved each time restart so I change to qbittorrent. I also tried linuxserver/rtorrent image but flood seems can't connect to it because I really don't know how to bind socket of rtorrent in 2 different containers.

**Comment by jesec** [comment, marked as answer]
Flood performs many filesystem operations locally. As a result, it needs direct filesystem access with shared filesystem context with torrent client. 

See [troubleshooting](https://github.com/jesec/flood#troubleshooting) section of README.

**Reply by lilws** [reply]
I did read that part but it still got troubles sometimes. I found that I need to set permission of these folder for everyone to rw on my NAS. By sh into docker container, I found that flood didn't have permission to access these folder. Thanks a lot.

**Reply by lilws** [reply]
> Flood performs many filesystem operations locally. As a result, it needs direct filesystem access with shared filesystem context with torrent client.
> 
> See [troubleshooting](https://github.com/jesec/flood#troubleshooting) section of README.

Hi, I spent sometimes with flood for qbittorrent and I just find a bug, I think. In qbittorrent everything act normal, but in flood, the location has a bug so that cause mediainfo through an error like that. For example: I watched a folder /downloads/torrents and when complete, it will move to /downloads/completed. During leeching the torrent, the location is right, but after it complete, the location miss the last letter and only show /downloads/complete. It suppose to be "completed" with the "d". Here the picture I save to software, but in flood it missed the "e".
![Screenshot 2021-01-28 214332](https://user-images.githubusercontent.com/12478478/106154549-4edb4d00-61b2-11eb-85a2-ec1ccc78073f.png)

The only fix I find so far is manually move it to another folder in flood then move it back.

**Reply by jesec** [reply]
See d345be43c08fe244bac8395029d55196f2616a13. Clearly qBittorrent API has a consistency problem when it comes to trailing slash.

**Reply by lilws** [reply]
great, hope flood new version will release soon on docker :)
